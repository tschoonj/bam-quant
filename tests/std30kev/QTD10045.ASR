File: C:\xrfdaten\2014KW30\chris\std30kev\std10045.spe
Start: 22.07.2014 10:14:35
Stop: 22.07.2014 10:16:15
BESSY: 298.5911835 298.6876519
IONI: 1.963265E-11 1.981348E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 67555
Run: 100000
Dead: 32
ICR: 113220
OCR: 0
Accumulated BESSY: 59710.4099947
Accumulated Ioni: 3.92672171000001E-09
N Accumulation: 200
Normfactor:      0.75394970
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Rh
exp_x 92.7203979492188 92.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  45   1     20.167   6051141.      2460.     107.04
