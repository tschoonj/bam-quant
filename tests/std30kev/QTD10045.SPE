$DATA:
0 2000
15074
10121
4813
1780
517
138
51
20
13
5
4
4
3
1
10
17
254
2603
3695
3721
3594
3485
3256
3218
3252
3141
2897
2858
2882
2682
2654
2621
2514
2443
2531
2447
2317
2359
2312
2347
2302
2316
2327
2294
2378
2345
2246
2361
2298
2343
2359
2435
2429
2379
2564
2462
2525
2591
2488
2459
2587
2612
2532
2612
2473
2504
2571
2513
2347
2372
2332
2337
2347
2304
2218
2076
2053
1905
1803
1757
1736
1564
1622
1515
1383
1382
1355
1328
1292
1307
1312
1141
1233
1201
1239
1163
1128
1124
1133
1114
1153
1148
1133
1105
1110
1025
979
912
881
836
803
843
733
757
726
742
702
682
719
656
648
634
644
663
648
574
585
624
553
588
554
584
548
561
550
566
518
510
488
518
532
518
511
514
503
524
490
434
481
494
468
452
471
473
485
494
510
463
438
442
489
455
401
437
436
438
445
424
445
439
446
429
415
413
429
419
411
428
399
391
432
416
410
414
380
435
379
404
395
371
408
408
408
347
370
364
385
358
367
394
394
350
379
373
353
348
364
361
378
386
372
372
387
388
362
360
317
373
377
360
344
376
364
335
354
327
363
361
373
346
366
344
357
342
332
362
334
357
367
319
306
337
337
335
343
321
331
315
338
294
326
368
324
340
368
330
347
300
356
304
341
310
346
321
341
329
427
389
401
400
356
371
337
298
346
330
323
326
340
305
318
270
323
318
291
311
305
297
302
303
334
349
321
313
317
336
319
320
320
307
316
320
281
284
305
317
276
293
309
332
355
341
371
473
531
638
645
710
662
612
575
499
391
352
323
295
280
290
305
260
289
298
289
299
299
275
304
299
267
296
301
276
307
313
313
319
314
342
332
355
343
336
335
356
324
315
274
295
304
268
283
305
296
291
275
295
310
307
309
313
280
298
293
318
310
289
297
294
291
273
291
275
246
268
284
277
291
270
289
289
252
273
253
275
319
305
249
298
315
297
292
281
281
297
294
309
289
265
269
293
280
274
305
286
264
270
257
253
256
289
276
280
247
281
283
307
292
284
301
277
318
276
233
249
276
284
296
249
246
266
274
256
284
271
271
276
273
258
258
259
236
254
286
291
300
316
303
315
307
341
336
299
294
300
305
312
295
279
315
331
304
284
256
296
292
271
296
251
261
256
250
262
236
260
259
254
236
252
245
273
233
247
246
256
275
266
255
246
252
254
259
255
256
243
279
273
258
278
245
261
260
242
249
252
249
271
283
286
306
314
258
266
285
261
231
264
273
267
260
253
273
261
258
286
306
298
284
283
315
299
298
300
303
353
370
333
348
417
392
436
397
386
388
391
350
339
362
360
330
324
280
309
288
295
275
261
256
267
234
227
246
272
245
264
264
260
253
257
249
252
257
260
257
265
248
274
264
243
238
259
257
206
284
287
256
260
281
262
261
275
264
256
226
267
255
274
271
256
247
249
251
262
249
293
275
275
239
281
267
280
246
274
288
275
329
287
303
305
266
275
287
298
297
256
302
304
313
315
319
324
345
340
344
338
329
323
319
315
320
287
300
276
292
290
271
285
308
304
316
306
301
304
310
291
303
280
283
314
275
257
305
297
260
310
285
278
291
266
281
275
308
305
288
263
297
291
284
290
291
268
306
278
296
296
289
303
271
288
292
283
310
280
283
284
253
296
279
255
287
287
304
273
288
285
292
312
314
311
303
314
327
291
338
329
317
335
284
330
346
331
343
349
331
332
340
322
393
342
384
352
364
325
373
397
361
363
382
385
377
399
353
341
382
436
387
384
399
415
403
383
415
397
434
442
533
592
710
796
1144
1587
2232
2957
4057
5368
6904
8386
9853
11037
12143
13172
13103
12714
11670
10570
9189
7741
6068
4545
3353
2430
1693
1248
912
746
578
508
474
425
388
370
396
429
407
390
413
383
428
379
390
445
417
423
407
446
406
429
448
409
418
431
440
400
458
445
426
436
439
415
435
441
451
436
448
468
497
480
433
424
496
491
459
485
475
502
491
499
462
508
482
493
502
517
523
488
512
495
504
527
519
523
576
594
585
664
714
800
954
1119
1269
1548
1741
2091
2279
2337
2476
2426
2356
2332
2068
1775
1555
1402
1281
1102
1027
954
958
957
951
982
985
961
936
895
930
905
866
869
892
852
938
896
934
903
959
1005
1105
1119
1170
1214
1261
1348
1313
1368
1419
1434
1411
1574
1457
1543
1480
1471
1527
1572
1557
1627
1621
1634
1722
1715
1781
1761
1767
1891
1846
1851
1871
1981
1950
1936
2047
2110
2093
2067
2098
2118
2239
2226
2304
2324
2346
2423
2343
2371
2315
2437
2464
2314
2290
2360
2373
2522
2295
2327
2435
2404
2593
2629
2691
2634
2865
2847
3109
3092
3267
3565
3788
4268
4985
6060
7705
10301
14061
19831
28253
39885
54591
73488
96132
123190
153019
185264
218137
252539
285124
315192
343730
364777
380533
387785
385193
373608
349779
318518
280234
239097
195476
155072
118996
87480
63013
43584
29487
19586
12531
8434
5703
4065
3103
2543
2148
1900
1586
1459
1422
1338
1315
1174
1171
1149
1040
1002
1020
1031
997
974
929
934
962
857
950
888
856
813
764
826
819
826
834
764
750
769
729
801
744
741
748
763
776
774
732
740
747
767
762
755
750
755
770
755
824
820
821
761
795
794
769
807
822
772
779
789
779
767
769
810
866
845
839
894
874
937
971
974
967
1009
954
1052
986
1055
1061
1149
1143
1232
1220
1360
1521
1597
1928
2323
3095
4016
5406
7877
10813
14947
20111
26454
33768
42141
50245
57964
65191
70522
73559
73737
71597
67456
61330
53350
44987
37251
29229
22600
17140
13278
10130
8008
7032
6639
7062
7712
8659
9730
10832
11619
12289
12238
12237
11431
10607
9516
8038
6531
5422
4040
3069
2116
1481
1077
769
483
325
223
171
126
94
71
70
62
70
53
67
52
44
49
66
60
44
52
55
46
45
43
39
42
32
36
31
36
35
40
35
35
35
33
25
28
26
40
29
30
22
34
28
21
27
35
21
33
28
25
25
16
18
25
24
21
17
25
19
20
18
14
17
26
13
14
15
17
22
8
18
18
21
19
17
23
23
20
16
16
15
15
10
16
16
23
20
13
14
23
13
19
17
19
18
14
12
13
6
13
10
7
17
14
9
12
5
6
12
9
18
12
5
9
15
15
9
9
12
7
8
12
10
11
6
4
9
14
9
13
9
14
8
14
11
11
10
10
16
12
12
10
15
13
15
16
11
13
15
18
14
10
13
10
11
14
9
12
7
18
14
10
8
15
11
5
11
4
10
11
8
8
8
16
9
8
5
13
11
7
11
14
6
8
9
9
7
11
6
6
6
10
9
10
11
12
16
4
10
5
5
14
7
7
12
6
8
9
10
8
8
6
6
10
7
14
14
7
7
11
6
15
2
13
6
7
12
10
9
9
10
8
3
5
6
7
12
7
7
1
8
10
8
7
8
16
7
9
7
12
10
12
6
6
13
7
8
6
7
7
7
10
6
8
6
10
11
7
7
12
8
7
7
7
14
10
7
11
8
12
3
5
5
8
5
4
2
5
16
4
10
7
4
7
6
8
7
4
12
11
10
12
10
11
15
9
7
4
8
4
4
7
7
10
9
12
4
5
5
6
8
3
8
7
12
7
8
9
6
4
7
5
6
4
8
13
10
7
13
12
8
10
4
4
9
9
7
7
2
13
8
8
11
11
8
10
12
5
8
8
11
10
6
10
1
5
7
12
7
7
12
12
8
10
7
5
8
4
9
7
11
16
8
10
9
10
6
8
7
13
12
12
10
9
13
8
17
5
6
7
13
11
7
11
7
8
7
11
10
7
8
10
10
9
2
6
7
14
7
8
12
7
10
9
9
7
8
10
9
4
17
5
14
8
8
8
14
8
2
9
7
11
9
15
10
10
7
8
10
11
13
8
12
13
13
14
14
10
15
13
10
9
15
8
11
13
8
12
12
17
16
11
14
13
14
14
7
13
8
18
19
9
8
14
10
12
10
15
8
14
14
11
15
5
5
11
13
14
13
17
14
24
13
14
7
18
6
21
20
11
10
20
19
10
21
20
14
14
6
16
10
15
25
13
15
16
8
14
10
18
11
10
12
23
18
17
14
14
22
13
12
14
17
22
13
10
8
19
13
16
15
14
10
11
11
14
19
10
14
23
18
21
23
26
21
22
25
19
18
27
24
27
34
27
36
28
22
37
28
28
19
26
34
31
34
19
29
30
31
29
20
20
35
41
35
33
36
31
37
52
38
50
54
93
87
83
93
95
123
116
130
136
130
132
133
159
133
143
138
117
129
91
87
68
75
55
55
53
35
34
22
12
17
12
15
12
9
7
6
10
14
5
3
12
12
18
11
7
8
9
5
11
7
11
3
5
4
8
6
5
8
3
6
8
10
10
6
9
6
10
11
9
8
12
5
7
7
8
10
6
6
7
8
10
12
8
7
9
13
17
18
11
17
21
14
19
19
26
16
30
26
31
39
22
23
21
40
31
34
33
34
28
29
34
32
34
36
34
21
21
24
20
25
16
36
20
25
23
24
21
27
32
37
38
29
38
46
37
52
50
49
50
38
31
43
31
32
38
53
40
38
35
32
35
33
23
41
32
32
31
37
31
35
35
35
33
40
32
38
43
26
41
34
37
37
38
41
32
43
29
36
37
36
44
36
34
42
34
40
37
36
34
45
53
37
32
43
44
44
57
57
61
47
75
82
96
118
119
146
193
229
233
292
372
424
521
657
744
811
1019
1143
1268
1372
1540
1652
1794
1957
2062
2125
2279
2300
2320
2352
2381
2313
2261
2175
2049
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10045.spe
Start: 22.07.2014 10:14:35
Stop: 22.07.2014 10:16:15
BESSY: 298.5911835 298.6876519
IONI: 1.963265E-11 1.981348E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 67555
Run: 100000
Dead: 32
ICR: 113220
OCR: 0
Accumulated BESSY: 59710.4099947
Accumulated Ioni: 3.92672171000001E-09
N Accumulation: 200
Normfactor:      0.75394970
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Rh
 
exp_x 92.7203979492188 92.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  45   1     20.167   6051141.      2460.     107.04
