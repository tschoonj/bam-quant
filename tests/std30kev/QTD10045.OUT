     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:51  บ   
     บ  Spectrum: QTD10045.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        4 iterations done    บ   
     บ  ChiSquare =   294.5   last change =   .00%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -24.4 ๑   100.0   บ     -24.4 ๑     1.0   บ   
     บ  GAIN (eV/ch)  บ     20.203 ๑   2.000   บ    20.203 ๑    .001   บ   
     บ det NOISE (eV) บ      121.3 ๑    40.0   บ     121.3 ๑     5.2   บ   
     บ  FANO factor   บ       .110 ๑    .050   บ      .110 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Rh-Ka                           6051141. ๑   2460.           บ   
     บ    KA1        20.216    .65377     3956038. ๑   1609.    116.2  บ   
     บ             1001.863    249.08         977.           9.47E-01  บ   
     บ    KA2        20.074    .34623     2095103. ๑    852.    108.1  บ   
     บ              994.835    248.41        1004.           9.50E-01  บ   
     บ                                                                 บ   
     บ  2 Rh-Kb                           1169477. ๑   1082.           บ   
     บ    KB1        22.724    .56918      665640. ๑    616.     37.0  บ   
     บ             1126.005    260.60         569.           8.81E-01  บ   
     บ    KB3        22.699    .29259      342172. ๑    318.     36.5  บ   
     บ             1124.767    260.49         574.           8.82E-01  บ   
     บ    KB2        23.173    .13824      161665. ๑    151.     30.8  บ   
     บ             1148.229    262.61         560.           8.67E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
